### ISSUE TITLE
Screen reader user is auto switched to browse mode when focus is moved on to a Sub Menu button in a dropdown

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4637]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** Keyboard 2.1.1

**Test Environment:**
- Test URL: https://clarity.design/documentation/dropdown
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. Open the test URL.
2. Go to angular dropdown and press it using enter or space key.
3. Navigate through the menu items and try to open the sub menu using right arrow key.

**Expected result:**
As NVDA enters the focus mode when the dropdown is opened, user can use arrow keys to navigate through the list. So user should be able to expand the sub menu using right arrow.

**Actual result:**
As soon as the focus reaches the sub menu, NVDA switches to browse mode and user is not able use arrow keys. Only Enter key or space key can be used at that point to expand the sub menu, which is not a standard behavior.
